### DISCUSSION TITLE
Error on running document_create_classifier

### DISCUSSION THREAD

**Discussion by sbrunner** [jonaswinkler/paperless-ng#688] [Support]
I just run: `document_create_classifier`
and I get:
```
[2021-03-03 17:40:54,151] [WARNING] [paperless.tasks] Classifier error: byte must be in range(0, 256) 
```
Does anyone know what I can do to fix that?

**Comment by jonaswinkler** [comment]
Oof. How many tags do you have?

**Reply by sbrunner** [reply]
Just 67, but about 500 correspondents, and 5863 documents.

**Reply by sbrunner** [reply]
It fails on a tag ID `336`...

**Reply by jonaswinkler** [reply]
Well yes, a clear case of me not being able to read documentation and some lack of foresight. :)

I'll get that fixed in the next version.
